docs/training-guide.md

```diff
@@ -1,17 +1,177 @@
+
+
+### Evaluation datasets
+- There might be statmt datasets available. For example `sacrebleu_wmt20`.
+  Run find-corpus to search using the [SacreBLEU tool](https://github.com/mjpost/sacrebleu):
+```
+python utils/find-corpus.py en ru sacrebleu
+```
+- Use some datasets for validation while training (`datasets.devtest` section) and others for evaluation (`datasets.test`).
+- Flores dataset is available for 100 languages, so it's always a good idea to add `flores_dev` for validation and `flores_devtest` for the final evaluation of the model.
+- Some OPUS and mtdata datasets provide dev and devtest versions, so it's a good idea to add them to evaluation.
+- Make sure that training, validation and evaluation datasets are different.
+
+```
+  # datasets to merge for validation while training
+  devtest:
+    - flores_dev
+    - sacrebleu_wmt19
+    - sacrebleu_wmt17
+  # datasets for evaluation
+  test:
+    - flores_devtest
+    - sacrebleu_wmt20
+    - sacrebleu_wmt18
+```
+
+### Monolingual corpus
+It is recommended to always use back-translations to augment training data and to use 
+monolingual corpus to augment data for decoding by the teachers, even for high resource lanugages.
+It will be especially useful for low-resource ones though. 
+The only limitation is probably available computational resources.
+
+Find monolingual data and add it to `datasets.mono-src` and `datasets.mono-trg`. 
+I usually use [News Crawl](https://data.statmt.org/news-crawl/) datasets from statmt 
+because they are relatively clean and we have an automatic downloading for them.
+```
+  # to be translated by the ensemble of teacher models
+  mono-src:
+    - news-crawl_news.2020
+    - news-crawl_news.2019
+    ...
+  # to be translated by the backward model to augment teacher corpus with back-translations
+  mono-trg:
+    - news-crawl_news.2020
+    - news-crawl_news.2019
+    ...
+```
+
+### Custom datasets
+
+It is also possible to use manually downloaded datasets with prefix `custom_<file_path>`.
+
+Find more details about the supported dataset importers [here](data.md).
+
+## 3. Configure data cleaning
+
+To use the default data cleaning pipeline set:
+```
+  use-opuscleaner: false
+```
+Make sure the language is present in [clean_parallel](https://github.com/mozilla/firefox-translations-training/tree/main/pipeline/clean/tools/clean_parallel.py#L19) script.
+
+For more advanced cleaning and for using OpusCleaner look at the [Data cleaning](cleaning.md) doc.
+
+### Bicleaner
+It is recommended to use Bicleaner ML models to filter noisy data. 
+Check that the bicleaner-ai model is [available](https://github.com/bitextor/bicleaner-ai-data/releases) 
+and add filtering thresholds to the config. 
+
+- `0.5` should be a good default value.
+- Noisier datasets like OpenSubtitles should have higher threshold. 
+- Set the threshold to `0` to skip cleaning entirely, for example for ParaCrawl dataset that comes already cleaned.
```

Comment:
```suggestion
- Set the threshold to `0` to skip cleaning entirely, for example for ParaCrawl dataset that comes already cleaned by bicleaner.
```

And we could add a reference to https://aclanthology.org/2022.lrec-1.87.pdf, section 4.2.2.